### PR TITLE
Scala 2.13.8

### PR DESCRIPTION
**Pull request by SethTisue**
for merge later this week

### CODE REVIEW

File: api/all.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ https://scala-ci.typesafe.com/artifactory/scala-integration/org/scala-lang/
 
 ## Previous releases
```
Comment (som-snytt) [inline api/all.md:59]: Trump would say, "Releases for Losers"

File: _overviews/jdk-compatibility/overview.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ For possible Scala issues, see the [jdk11](https://github.com/scala/bug/labels/j
 
 Early access builds of JDK 18, a non-LTS release, are already available.
 
-Initial support for JDK 18 is included in Scala 2.13.7 and 2.12.15.
+Initial support for JDK 18 was included in Scala 2.13.7 and 2.12.15.
```
Comment (som-snytt) [inline _overviews/jdk-compatibility/overview.md:86]: Of course, it is still included. We're so proud of English until we encounter certain modes.

Comment (SethTisue) [reply _overviews/jdk-compatibility/overview.md:86]: I also considered "has been included since", but 🤷

Comment (som-snytt) [reply _overviews/jdk-compatibility/overview.md:86]: for 2022, I predict emoticons will experience retro hipster cachet.

**Pull request merged**